src/html/authorize.html.template

```diff
@@ -34,7 +34,7 @@
       <ha-authorize><p>Initializing</p></ha-authorize>
     </div>
     <% if (!latestBuild) { %>
-      <script src="/static/custom-elements-es5-adapter.js"></script>
+      <script src="/static/custom-elements-es5-adapter"></script>
```

Comment:
I'm not sure about these `.template` file modifications I made either. Things seem to run fine in my testing...?

Reply:
Yeah revert them. 

When you tested locally, did you navigate to the es5 version of authorize ?